includes/Story_Post_Type.php

```diff
@@ -281,6 +281,10 @@ public static function admin_enqueue_scripts( $hook ) {
 			$max_upload_size = 0;
 		}
 
+		// @todo Make filterable like in Story_Post_Type::get_schemaorg_metadata()
+		$publisher      = get_option( 'blogname' );
```

Comment:
I think that's the same question as I had above. If we had a way to merge `<html><head>`, we could always keep these kind of metadata props on the server and simply merge them into the final HTML.

Reply:
Yep, same question. It's just not `<head>` only that would be affected